README.md

```diff
@@ -300,6 +316,44 @@ class AuthenticatedController < ApplicationController
 end
 ```
 
+### ScopesVerification
+The `ShopifyApp::ScopesVerification` concern helps merchant grant new access scopes requested by the app. The concern compares the current access scopes granted by the merchant and compares it with the scopes requested by the app. If there is a mismatch in configuration, the merchant is redirected to login via OAuth and grant the net new scopes.
+
+This requires the app to maintain a record of the current access scopes per merchant. This can be done by defining the template class methods in the model that includes `ShopifyApp::ShopSessionStorage`:
+```ruby
+class Shop < ActiveRecord::Base
+  include ShopifyApp::ShopSessionStorage
+
+  def self.update_merchant_scopes(shop, scopes)
```

Comment:
While I like this pattern and how it allows us to add this behaviour only if the migration is set up, I wonder if we couldn't use `scopes=` and `scopes` as our method names, which feels like it might be easier to understand. Would that work at all?

Reply:
I just pushed up a change introducing the `scopes` and `scopes=` instance methods for the Shop model.